### ISSUE TITLE
Noscript element

### ISSUE THREAD

**Issue by r-leyshon** [banksad/returns_to_education#3]
**<h3>WCAG2.1 accessibility issues</h2>**

**Type**: Alert

**Source**: streamlit.io

**Short desc**: Noscript element

**Long desc**: A fall back in the case of JavaScript being disabled warns that JS is required to run the app.

What It Means
A `<noscript>` element is present.
Why It Matters
Content within `<noscript>` is presented if JavaScript is disabled. Because nearly all users (including users of screen readers and other assistive technologies) have JavaScript enabled, `<noscript>` cannot be used to provide an accessible version of inaccessible scripted content.
How to Fix It
Ensure that scripted content is accessible. The `<noscript>` content will be presented to very few users, but must be accessible if used.
The Algorithm... in English
A `<noscript>` element is present.

**WCAG ref**:
None

**Code ref**:

```
<noscript>
You need to enable JavaScript to run this app.
</noscript>
```
